src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,16 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name = "hibernate.dialect">org.hibernate.dialect.MySQLDialect</property>
+        <property name = "connection.url">jdbc:mysql://localhost:3306/test</property>
+        <property name = "connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name = "connection.username">root</property>
+        <property name = "connection.password">rfrfirf</property>
+        <property name = "show_sql">true</property>
+        <property name = "hbm2ddl.auto">create-drop</property>
+
+        <mapping class ="mate.academy.model.Movie"/>
+        <mapping class ="mate.academy.model.CinemaHall"/>
+        <mapping class ="mate.academy.model.MovieSession"/>
+        <mapping class ="mate.academy.model.User"/>
+    </session-factory>
+</hibernate-configuration>
```

Comment:
add one empty line, when you see this in pr
![image](https://github.com/mate-academy/hibernate-user-service-hw/assets/49767049/ac5138de-bf18-4983-a746-39b0bf2b5907)